Add `Ember Data flexibility` section (#240)
ember-learn/guides-source: guides/v3.5.0/models/index.md

[Specifying a Route's Model](../routing/specifying-a-routes-model/)
guide.

At first, using Ember Data may feel different than the way you're used
to writing JavaScript applications. Many developers are familiar with
using AJAX to fetch raw JSON data from an endpoint, which may appear
easy at first. Over time, however, complexity leaks out into your
application code, making it hard to maintain.

With Ember Data, managing models as your application grows becomes both

> simple _and_ easy.

Reviewer: ```suggestion
simpler _and_ easier.
```
It can still be hard to use Ember Data, sometimes. I think maybe the -er is a bit more realistic and kinder to new learners.

Once you have an understanding of Ember Data, you will have a much
better way to manage the complexity of data loading in your application.

> This will allow your code to evolve without becoming a mess.

Reviewer: ```suggestion
This will allow your code to evolve and grow, with better maintainability.
```

Agreed, if we can not use the "mess" language, it's better. Some of my past writing drew criticism for a similar statement and I think people had good points.
